packages/deployments/subgraph/src/amarok-runtime-staging/mapping.ts

```diff
@@ -251,7 +252,13 @@ export function handleExecuted(event: Executed): void {
       router.isActive = true;
       router.save();
     }
+
     routers.push(router.id);
+
+    // Update router's liquidity
+    const assetBalance = getOrCreateAssetBalance(event.params.args.local, event.params.args.routers[i]);
+    assetBalance.amount = assetBalance.amount.minus(amount.div(BigInt.fromI32(num)));
```

Comment:
When we are doing multipath, are all the routers contributing the same amount? Noticed we don't really ask for the amount in router signature.

Reply:
Yes. contributing the same amount